USFMToolsSharp/USFMParser.cs

```diff
@@ -72,7 +74,24 @@ private List<Marker> TokenizeFromString(string input)
                 result.marker.Position = match.Index;
                 output.Add(result.marker);
 
-                if (!string.IsNullOrWhiteSpace(result.remainingText))
+                //Conditions for the spaces between tags to be included as whitespace in a verse
+                var vMarker = new VMarker();

```

Comment:
I think we probably want this to work on more then just one marker type and not just VMarker. I'm thinking of some of the following.
- Adding a property to the marker to whether or not it should preserve space
- I'm fairly certain it is just the space between end and beginning markers that we need to preserve. I think we could handle this more generically.

And I'm fairly certain I had something else and I'm hoping I remember it on Monday.

Reply:
Yeah, we have quite a lot of stuff that needs to be done in the whitespace area, I'm going to be coming back to this with more time in a couple weeks. So we'll give it a shot then.